internetarchive/openlibrary
# Create usergroup that cannot edit

Closes #6500


Creates `read-only` usergroup.  Edits by members of this group will fail our spam checks (unless the member is also an admin or librarian).

### Technical

Before deploying this code to testing or production, `/usergroup/read-only` must be created.  I suspect that all spam checks will fail if this isn't done.  ~Labeling this PR as `Needs: Special Deploy` in order to bring attention to this.~

__Edit:__ `/usergroup/read-only` has been created in production.

### Testing

These instructions assume that you are testing locally.  We will be adding local user `openlibrary` to the new usergroup.
1. Login as user `admin`. Remove user `openlibrary` from usergroup `admin`.
2. Add `/people/openlibrary` to the `/usergroup/read-only` group.
3. While still logged in as `admin`, navigate to a book page, and make a test edit (don't include spam words in the edit). The edit should be successful.
4. Log in as `openlibrary`.  Navigate to a book page and attempt to make an edit.  The edit should fail.

### Screenshot


### Stakeholders

@mekarpeles @seabelis
